src/marqo/tensor_search/tensor_search.py

```diff
@@ -297,7 +299,8 @@ def _add_documents_unstructured(config: Config, add_docs_params: AddDocsParams,
                             split_overlap = marqo_index.text_preprocessing.split_overlap
                             content_chunks: List[str] = text_processor.split_text(field_content, split_by=split_by,
                                                                                   split_length=split_length,
-                                                                                  split_overlap=split_overlap)
+                                                                                  split_overlap=split_overlap,
+                                                                                  prefix=text_chunk_prefix)
```

Comment:
we want to be able to have content_chunks (with prefix) and text_chunks (without prefix). This way we can store the text in vespa with no prefix, but the embeddings will be affected by it. That's why it's better to have the prefixing not be in the `split_text` function.

Reply:
I think with what we did before, the text chunks are stored normally already with only the prefix going to the embeddings directly. 

But you are right that it's better to separate the prefixing logic from `split_text`. I have since just wrapped the output of `split_text` with `prefix_text_chunks` to get the prefixed version according to your suggestion. Thanks!